src/templates/language_selector.html

```diff
@@ -1,4 +1,4 @@
-<ul>
+<div id="language-selector">
```

Comment:
Before it said `<ul>`, but it was removed here: https://github.com/dimagi/Vellum/pull/587/files#diff-e5b3eca8407b45b132e0147b8392192cL1421 so it was just `<li>...`

That would make it very hard to remove here: https://github.com/dimagi/Vellum/pull/587/files#diff-e5b3eca8407b45b132e0147b8392192cR1421 I didn't see any visual differences by wrapping it in a div, although to be honest I wasn't sure if:
```
<ul>
  <li>
  <div>
    <li>
  </div>
</ul>
``` was a correct pattern (this is inserted into a list which is why it doesn't need `<ul>`